AirBench/scripts/index.js

```diff
@@ -0,0 +1,67 @@
+﻿(async () => {
+    const baseUrl = "http://localhost:52346/api/map";
```

Comment:
This is a nitpick, but the variable name `baseUrl` suggests that it's everything up to the resource name, so `http://localhost:52346/api/`.